tests/__fixtures__/testWithTranslationHOC/classComponent.js

```diff
@@ -2,28 +2,40 @@ import * as React from 'react';
 import { withTranslation } from 'react-i18next';
 import * as ReactI18Next from 'react-i18next';
 
+function ensureT(message, t) {
```

Comment:
I don't think having this function declared here is really relevant since this code is just parsed statically. You can probably remove it.